src/test/java/org/sonatype/nexus/ci/util/NxrmUtilTest.groovy

```diff
@@ -155,20 +159,38 @@ class NxrmUtilTest
       listBoxModel.get(2).value == repositories.get(4).id
   }
 
+  @Unroll
+  def 'isVersion(#id, #version) == #expected'() {
+    setup:
+      createNxrmConfig('nxrm2')
+      createNxrmConfig('nxrm3')
+
+    expect:
+      NxrmUtil.isVersion(id, version) == expected
+
+    where:
+      id      | version | expected
+      'nxrm2' | NEXUS_2 | true
+      'nxrm2' | NEXUS_3 | false
+      'nxrm3' | NEXUS_3 | true
+      'nxrm3' | NEXUS_2 | false
+  }
+
   //get a config, defaults to nxrm3
-  NxrmConfiguration createNxrmConfig(String id = 'nxrm3') {
-    def configurationList = []
+  private NxrmConfiguration createNxrmConfig(final String id = 'nxrm3') {
+    def globalConfiguration = jenkins.getInstance().getDescriptorByType(GlobalNexusConfiguration)
+    if (!globalConfiguration.nxrmConfigs) {
+      globalConfiguration.nxrmConfigs = []
+    }
+
     def nxrmConfiguration
     if (id == 'nxrm2') {
       nxrmConfiguration = new Nxrm2Configuration(id, "internal${id}", 'displayName', 'http://foo.com', 'credentialsId')
     } else {
       nxrmConfiguration = new Nxrm3Configuration(id, "internal${id}", 'displayName', 'http://foo.com', 'credentialsId')
     }
 
-    configurationList <<  nxrmConfiguration
-
-    def globalConfiguration = jenkins.getInstance().getDescriptorByType(GlobalNexusConfiguration)
-    globalConfiguration.nxrmConfigs = configurationList
+    globalConfiguration.nxrmConfigs << nxrmConfiguration
     globalConfiguration.save()
```

Comment:
Nice clean up and conslidation, will boyscout the others to make them consistent.